packages/block-editor/src/hooks/padding.js

```diff
@@ -65,13 +65,23 @@ export function PaddingEdit( props ) {
 		} );
 	};
 
+	const onReset = () => {
```

Comment:
Shouldn't it accept the `defaultValue` as function param to account for the case when the style attribute is not provided with defaults?

Reply:
Done